GTMAppAuthSwift/Sources/KeychainStore.swift

```diff
@@ -79,6 +79,8 @@ public final class KeychainStore: NSObject {
     self.itemName = itemName
     self.keychainAttributes = keychainAttributes
     self.keychainHelper = keychainHelper
+
+    NSKeyedUnarchiver.setClass(AuthState.self, forClassName: "GTMAppAuthFetcherAuthorization")
```

Comment:
Any reason _not_ to set the translation mapping globally for our use case?

Reply:
I think it makes sense to restrict the scope of this change. I've moved onto using `NSKeyed(Un)Archiver` instances.